### PR TITLE
Upgrade to Devise 3.2.1

### PR DESCRIPTION
**Pull request by bishboria**
The last instalment of Upgrade Devise Saga

### CODE REVIEW

File: test/integration/password_expiry_test.rb [hunk 1]
```diff
@@ -25,12 +25,12 @@ class PassphraseExpiryTest < ActionDispatch::IntegrationTest
     end
 
     should "remember where the user was trying to get to before the password reset" do
-      visit "/users/#{@user.id}/edit?arbitrary=1"
+      visit "/users/#{@user.id}/edit"
 
       signin(@user)
       reset_expired_passphrase(@user.password, @new_password, @new_password)
 
-      assert_current_url "/users/#{@user.id}/edit?arbitrary=1"
```
Comment (boffbowsh) [inline test/integration/password_expiry_test.rb]: Why was this changed?

**Comment by boffbowsh** [comment]
Looks good to me :+1:

File: test/integration/password_expiry_test.rb (hunk 1 above)
Comment (bishboria) [inline test/integration/password_expiry_test.rb]: you might've missed the commit note, devise changed how it deals with `user_return_to` in the session. Here's the change: https://github.com/plataformatec/devise/compare/v3.1.0...v3.2.1#diff-69e4dec55d542382a2998bef45d66401R34
